imap_processing/cdf/defaults.py

```diff
@@ -28,4 +28,5 @@ class GlobalConstants:
     INT_MAXVAL = np.iinfo(np.int64).max
     DOUBLE_FILLVAL = np.float64(-1.0e31)
     MIN_EPOCH = -315575942816000000
+    # TODO: This is set to 2029, should be further out. Determine the correct value.
```

Comment:
good catch!

Reply:
All Andriy! I wasn't actually sure how to calculate a different Epoch, which is why I didn't change it (or what a good number would be)

Reply:
Might be my bad, I believe I took an example number on the ISTP website and assumed it was the "default" VALIDMAX attribute.  MMS uses "2100-01-01T00:00:00" as their VALIDMAX so I vote we use that too.  We are using MMS's VALIDMIN, which is "1900-01-01T00:00:00".  

That would make the value: 

3155630469184000000

Reply:
Seems like a good number, I'll update it!